ssntp/ssntp_test.go

```diff
@@ -842,6 +861,16 @@ func roleToCert(role uint32) string {
 	return testCertUnknown
 }
 
+func getFileConfigURI() string {
+	filename := tempCertPath + "/configuration.yaml"
```

Comment:
Please define a tempConfigPath filesystem path for the configuration YAML.
And remove it from TestMain() as well.